spec/requests/api/v1/organizations_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+
+RSpec.describe "/api/v1/organizations", type: :request do
+  # This should return the minimal set of attributes required to create a valid
```

Comment:
Este archivo deberia quedar vacio hasta que se cree un endpoint
```suggestion
  RSpec.describe "/api/v1/organizations", type: :request do
  end
```

Reply:
Entendido